### PR TITLE
Fix wording

### PR DESCRIPTION
**Pull request by viroulep**
As per @Luis-J-Ianez comment.

### CODE REVIEW

**Comment by DanielEgdal** [comment]
I think that it would fix the issues, if there were a check box (like the one with "use the wca page for registration", and then this would only appear if that is checked), which would determine if on-site registration would be possible. Then if that was checked, then it could display the suggested, with online registration, then it would also specify the special points of on-site registration. (So that would be, that it would only be possible to register on-site, if the competitor limit wasn't reached).
If the box then wouldn't be checked, then it would only display "registration period", and not the word online.

I hope what I wrote makes sense to you, I can see it being a bit confusing. :)

**Comment by Luis-J-Ianez** [comment]
What about simply adding a clarification? Something like:

Online registration period: from X to Y. Check in the rest of the competition details whether more registration periods and/or systems will be available (for example, on the spot registration).

**Comment by viroulep** [comment]
I like it!
I've currently updated the PR with this last suggestion, but feel free to suggest alternative wording.

File: WcaOnRails/config/locales/en.yml [hunk 1]
```diff
@@ -896,8 +896,8 @@ en:
       hide_requirements: "Hide registration requirements."
       #context: when displaying the registration opening/closing date
       registration_period:
-        label: "Registration period"
-        range_html: "From %{start_date_and_time} to %{end_date_and_time}."
+        label: "Online registration period"
+        range_html: "From %{start_date_and_time} to %{end_date_and_time}. Check in the rest of the competition details whether more registration periods and/or systems will be available (for example, on the spot registration)."
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: The current wording feels a bit awkward to me. Can we rewrite this a bit? How about:

> Online registration will be open from <start> to <finish>. Other forms of registration may be allowed (such as registration at the door). Be sure to read through all the competition details to find out.

I'd also change the label from "Online registration period" to just "Registration period", as we're already clarifying what it means here.

Comment (Luis-J-Ianez) [reply WcaOnRails/config/locales/en.yml]: Good to me :)

**Comment by viroulep** [comment]
The PR has been sitting here for a while and the wording used pleased everyone, so I'm merging this!

**Comment by Mollerz** [comment]
Hi all, please see my issue linked here: https://github.com/thewca/worldcubeassociation.org/issues/2873

I think this can be construed as confusing and I would highly recommend we only show the opening and closing dates and nothing else.

**Comment by Luis-J-Ianez** [comment]
How does this address my concerns that a regular visitor can be misled and think the online registration is the only possible registration? Somehow it must be clear that on the spot registrations are possible in some cases.

**Comment by Mollerz** [comment]
The registration requirements are written **directly below** in every case. All this should state is that the registration opens and closes. I think it's far worse if people turn up expecting to be able to register on the door than people not showing up at all.

https://gyazo.com/2f73d6f0c8149fbe338eb25514d1cac4

If you accept registration on the door as well, it's the first thing that you can write in the registration requirements and bold and highlight it and make it really clear.

**Comment by Luis-J-Ianez** [comment]
OK, you convinced me. :) Thanks. We can move forward on this.

**Pull request merged**